community/settings.py

```diff
@@ -64,7 +65,6 @@
 
 WSGI_APPLICATION = 'community.wsgi.application'
 
-
```

Comment:
leave this line also; someone else can do PEP cleanup

Reply:
@sks444 You forgot this 😄